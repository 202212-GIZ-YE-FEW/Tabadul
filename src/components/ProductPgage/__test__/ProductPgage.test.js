import renderer from "react-test-renderer";

import Product from "../ProductPage";

jest.mock("next/router", () => ({
    useRouter() {
        return {
            pathname: "",
            // ... whatever else you you call on `router`
        };
    },
}));

it("renders correctly", () => {
    const tree = renderer.create(<Product />).toJSON();
    expect(tree).toMatchSnapshot();
});
