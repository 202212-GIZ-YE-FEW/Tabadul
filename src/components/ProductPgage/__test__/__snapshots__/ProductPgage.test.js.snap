// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="css-ol394g"
>
  <div
    className="css-hsfvbr"
  >
    <div
      className="css-1viuenm"
    >
      <div
        className="active-image-container"
      >
        <img
          alt="Product"
          className="active-image"
          src=""
        />
      </div>
      <ul
        className="active css-vcznx6"
        style={
          Object {
            "paddingRight": "50px",
          }
        }
      >
        <div
          className="css-oh7kvy"
          onClick={[Function]}
        >
          <img
            alt="Previous"
            src="/images/left.png"
          />
        </div>
        <div
          className="css-1rayqgy"
          onClick={[Function]}
        >
          <img
            alt="Next"
            src="/images/right.png"
          />
        </div>
      </ul>
    </div>
    <div
      className="css-c7somy"
    >
      <h1
        className="productName"
      />
      <p />
      <div
        className="css-1p9w2x1"
      >
        <img
          alt="Location Logo"
          className="css-4ksxhv"
          src="/images/location.png"
        />
      </div>
      <div
        className="css-oa1um5"
      >
        <img
          alt="Info"
          className="css-1l8w7ag"
          src="/images/info.png"
        />
        <div>
          <p>
            name
             : 
          </p>
          <p>
            phone
             : 
          </p>
          <p>
            Email
             : 
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
